tests/zfs-tests/tests/functional/cli_root/zpool_status/zpool_status_-c_fullpath.ksh

```diff
@@ -0,0 +1,71 @@
+#!/bin/ksh -p
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright (c) 2017 by Lawrence Livermore National Security, LLC.
+#
+
+# DESCRIPTION:
+# Verify zpool status command mode (-c) works with full path to script.
+#
+# STRATEGY:
+# 1. Make a simple script that echos a key value pair in /var/tmp
+# 2. Make sure it can be run with -c via it's full path
+# 3. Remove the script we created
+
+. $STF_SUITE/include/libtest.shlib
+. $STF_SUITE/include/zpool_script.shlib
+
+verify_runnable "both"
+
+typeset USER_SCRIPT="/var/tmp/userscript"
+
+function cleanup
+{
+	if [[ -f "$USER_SCRIPT" ]]; then
+		rm -f "$USER_SCRIPT"
+	fi
```

Comment:
If you use `rm -f` to remove it you really don't need to verify it exists.  Maybe just `log_must rm -f "$USER_SCRIPT"`.